### PR TITLE
Enable creating a PR when running the Crowdin translation sync workflow

### LINKED ISSUE #13102: Make the `Sync Translations` workflow create its own PR
Context:
To sync translations, we run [this github workflow](https://github.com/ankidroid/Anki-Android/actions/workflows/sync_translations.yml). Then we have to create a PR from the `i8n_sync` branch and review it. But as the reviewer needs to create the PR, they may not be able to review/merge the PR because they created it themselves and the repo doesn't allow (for safety reasons) most reviewers to approve their own PRs.

Motivation:
- Spare the time of creating the PR
- Avoid the issue of a reviewer having to create the PR and so not being able to merge the PR because they created it themselves

Task:
After running the workflow, create automatically the translations PR with title of "Updated strings from Crowdin YYYYMMDDHHmm` (YYYYMMDDHHmm being a date format)

### PR DESCRIPTION
**Pull request by lukstbit**
## Purpose / Description

Creates a PR for strings changes. I added an extra step which verifies that there are changes before actually creating the PR to better separate the work. @BrayanDSO The format of the date is `2023/01/13 08:38` (so UTC time), let me know if this is ok(I assumed that _YYYYMMDDHHmm_ is a placeholder for a date format and not the actual format requested). I added basic strings for title and body, these of course, can be changed as needed.

## Fixes
Fixes #13102 

## How Has This Been Tested?

Tested it on my fork and it does create the requested PR(couldn't test the no strings changes scenario, but I don't see why this wouldn't work).

## Checklist

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by david-allison — APPROVED** [review]
LGTM. thanks

File: .github/workflows/sync_translations.yml [hunk 1]
```diff
@@ -52,3 +53,37 @@ jobs:
...
+            echo "HAS_CHANGES=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create PR for strings changes if needed
+        if: ${{ steps.tr_check.outputs.HAS_CHANGES }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const now = new Date();
+            const formattedDate =
```
[the remark this answers is not in the document]
Comment (lukstbit) [reply .github/workflows/sync_translations.yml:75]: Done!

**Comment by BrayanDSO** [comment]
(I assumed that YYYYMMDDHHmm is a placeholder for a date format and not the actual format requested).

Yes, it was, and I like the date format you chose

**Review by BrayanDSO — APPROVED** [review]
LGTM, and it worked on [your fork](https://github.com/lukstbit/Anki-Android/pull/27)

~~I'll left it as a Pending merge, as you've asked Mike's review and he understands gh workflows lightyears better than me~~

edit: Actually, this may take some time, so I'll just do a test drive

**Pull request merged**